### DISCUSSION TITLE
Making changes to webpack.config file?

### DISCUSSION THREAD

**Discussion by chrisedhog** [vercel/community#81] [Help] [answered]
Hi, I've run into an issue - I need to use the Buffer package in my react app, however Webpack 5 doesn't include it. The solution has been to manually include this by editing the webpack.config.js file within the node packages folder, following [this guide here](https://viglucci.io/how-to-polyfill-buffer-with-webpack-5).  The solution work, however it of course only works locally. I'm stuck as to how I can make the same changes for the version that is deployed on Vercel.

Any advice on how I can fix this?

**Comment by Pieparker** [comment]
Not my area of expertise however it seems like this is a [common issue](https://github.com/diegomura/react-pdf/issues/1029) around the web for Webpack 5 users. Suggestions in that link, [or perhaps this issue](https://github.com/webpack/changelog-v5/issues/10) may help?

This would be a good question for the Webpack team, if issues persist!

**Comment by barelyhuman** [comment, marked as answer]
While a hacky way to do it , https://www.npmjs.com/package/patch-package is what you can use here. 

It'll create a diff between your modified version and the actual released version, thus running the patch after installation, so it should work on both platforms as expected, you are better off re-pinging on the developers for a more graceful solution though.
